commercetools-models/src/test/java/io/sphere/sdk/subscriptions/commands/AbstractQueueIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+package io.sphere.sdk.subscriptions.commands;
+
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+import static io.sphere.sdk.subscriptions.SubscriptionFixtures.*;
+import static org.junit.Assume.assumeNotNull;
+
+
+public abstract class AbstractQueueIntegrationTest extends IntegrationTest {
```

Comment:
In my last review I suggested to use an `AzureSBUtils` class instead of having this abstract test class. What do you think about? May be let's have a chat about it tomorrow.